### DISCUSSION TITLE
How can I Stop PowerShell reminders every time there is a new release?

### DISCUSSION THREAD

**Discussion by ColumbusAI** [PowerShell/PowerShell#24200] [General]
This gets annoying and while I appreciate the robust development, I'd like to have the option to turn it off if I don't want to upgrade every incremental release and want to wait for major version releases.

**Comment by ThomasNieto** [comment]
https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_update_notifications?view=powershell-7.4

**Reply by ColumbusAI** [reply]
Thank you!
